Add the accessPackageResourceEnvironment resource type (#11273)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/connectioninfo.md

---
title: "connectionInfo resource type"
description: "The connectionInfo object defines the connection information that is used to communicate with a resource."
author: "hanki-microsoft"
localization_priority: Normal
ms.prod: "microsoft-identity-platform"
doc_type: resourcePageType
---

# connectionInfo resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

The connectionInfo object defines the resource locator that is used to communicate with a resource in Azure AD Entitlement Management.

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |url|String|The endpoint that is used by ELM to communicate with the access package resource.|

Reviewer: "ELM" is an internal code name from prior to GA.  This should say "Entitlement Management"

---
title: "connectionInfo resource type"
description: "The connectionInfo object holds the connection information which will be used by ELM to communicate with a resource."
author: "markwahl-msft"
localization_priority: Normal
ms.prod: "microsoft-identity-platform"
doc_type: resourcePageType
---

# connectionInfo resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> The connectionInfo object holds the resource locator which will be used by Entitlement Management (ELM) to communicate with a resource.

Reviewer 2: ```suggestion
The connectionInfo object defines the resource locator that is used to communicate with a resource in Azure AD Entitlement Management.
```
